packages/google_scc/data_stream/asset/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,180 @@
+---
+description: Pipeline for processing Asset logs.
+processors:
+  - set:
+      field: ecs.version
+      tag: set_ecs_version
+      value: 8.8.0
+  - set:
+      field: event.kind
+      tag: set_event_kind
+      value: event
+  - set:
+      field: event.category
+      tag: set_event_category
+      value: [host]
+  - set:
+      field: event.type
+      tag: set_event_type
+      value: [info]
+  - rename:
+      field: message
+      target_field: event.original
+      tag: rename_message
+      ignore_missing: true
+  - json:
+      field: event.original
+      tag: 'json_decoding'
+      target_field: json
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag fail-{{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
```

```suggestion
            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
```
The "fail-{{{_ingest.on_failure_processor_tag}}" tag is not added. Also below, though this is an issue under discussion (https://github.com/elastic/integrations/pull/6593#discussion_r1245869962).